CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+-   Fix `type` props forwarding on `TextField` component.
+-   Remove useImage hook
+-   Refactor `Thumbnail` component to have a better logic to handle image states
```

Comment:
I think it would be clearer like this (the useImage hook is not exported and it's more of a fix rather than a refactor) 
```suggestion
-   Fix `Thumbnail` component loading and error state handling
```